templates/systemd-prometheus-omero-exporter-service-debian.j2

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=prometheus-omero-exporter
+
+[Service]
+User={{ omero_prometheus_exporter_system_user }}
+EnvironmentFile=-/etc/prometheus-omero-exporter
```

Comment:
Since these are templates already, should the value of `EnvironmentFile` become a variable itself?

`defaults/main.yml` could define a dictionary of the values of this environment file depending on the operating system family

Reply:
fixed in https://github.com/ome/ansible-role-omero-prometheus-exporter/pull/14/commits/8df2fe974d6f0ea28a3d2917cb7c1c3b0ba38d03